tests/DataFactories/ScheduledTask.php

```diff
@@ -8,17 +8,44 @@
 use MailPoet\Cron\Workers\SubscriberLinkTokens;
 use MailPoet\Cron\Workers\UnsubscribeTokens;
 use MailPoet\Cron\Workers\WooCommercePastOrders;
+use MailPoet\DI\ContainerWrapper;
+use MailPoet\Entities\ScheduledTaskEntity;
 use MailPoet\WP\Functions as WPFunctions;
 use MailPoetVendor\Carbon\Carbon;
+use MailPoetVendor\Doctrine\ORM\EntityManager;
 
 class ScheduledTask {
+  private $diContainer;
+  private $entityManager;
```

Comment:
It would be nice to have doctypes for these properties.

Reply:
Done